RELEASING.md

```diff
@@ -32,23 +32,15 @@ tools must be installed.
 Release Steps
 -------------
 
-### Preparing Documentation
+### Release Documentation update
 
-Update version-sensitive files in the root and documentation sources
-in the `docs` subdirectory:
-
-* `docs/changelog.txt`
-* `docs/versions.txt`
-* `docs/upgrade.txt`
-
-Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to
-build a set of docs for this release and copy them to the cloned
-`docs.janusgraph.org` repo which you will update later.
-
-You may also need to update the following files in the main repo for any new
-or updated dependencies:
-
-* `NOTICE.txt`
+1. Update version-sensitive files in the root and documentation sources in the `docs` subdirectory: 
+    * `docs/changelog.md`
+2. Update the configuration reference: `mvn install -DskipTests=true -pl janusgraph-doc -am`
+3. Build documentation: `mkdocs build`
+4. Zip documentation: `$ zip janusgraph-${JANUSGRAPH_VERSION}-hadoop2-doc.zip site`
+4. You may also need to update the following files in the main repo for any new or updated dependencies:
+    * `NOTICE.txt`
```

Comment:
Please also add a section to `BUILDING.md` on how the docs can be build and shown in preview so it's possible to view the docs locally. The docs we currently have in [docs.janusgraph.org](https://github.com/JanusGraph/docs.janusgraph.org) won't be useful anymore with this change. So, we need new instructions for the MkDocs based docs.